src/test/java/org/opensearch/security/dlic/rest/api/RolesApiTest.java

```diff
@@ -532,26 +532,32 @@ public void checkNullElementsInArray() throws Exception{
         String body = FileHelper.loadFile("restapi/roles_null_array_element_cluster_permissions.json");
         HttpResponse response = rh.executePutRequest(ENDPOINT + "/roles/opendistro_security_role_starfleet", body, new Header[0]);
         Assert.assertEquals(HttpStatus.SC_BAD_REQUEST, response.getStatusCode());
+        Assert.assertTrue(response.getBody().contains("`null` is not allowed as json array element"));
 
         body = FileHelper.loadFile("restapi/roles_null_array_element_index_permissions.json");
         response = rh.executePutRequest(ENDPOINT+ "/roles/opendistro_security_role_starfleet", body, new Header[0]);
         Assert.assertEquals(HttpStatus.SC_BAD_REQUEST, response.getStatusCode());
+        Assert.assertTrue(response.getBody().contains("`null` is not allowed as json array element"));
 
         body = FileHelper.loadFile("restapi/roles_null_array_element_tenant_permissions.json");
         response = rh.executePutRequest(ENDPOINT + "/roles/opendistro_security_role_starfleet", body, new Header[0]);
         Assert.assertEquals(HttpStatus.SC_BAD_REQUEST, response.getStatusCode());
+        Assert.assertTrue(response.getBody().contains("`null` is not allowed as json array element"));
```

Comment:
Lots of repitition. May be put as a variable and use it

Reply:
Do you mean variable for `null is not allowed as json array element`?

Reply:
Done

Reply:
Use `ErrorType.NULL_ARRAY_ELEMENT`.